package.json

```diff
@@ -12,9 +12,11 @@
     "start": "wp-scripts start ./src/assets/js/main.js --output-path=./unlock-wordpress-plugin/assets/build/js",
     "start:blocks": "wp-scripts start ./src/assets/js/blocks.js --output-path=./unlock-wordpress-plugin/assets/build/js",
     "build:blocks": "wp-scripts build ./src/assets/js/blocks.js --output-path=./unlock-wordpress-plugin/assets/build/js",
+    "start:fullpostpage": "wp-scripts start ./src/assets/js/fullpostpage.js --output-path=./unlock-wordpress-plugin/assets/build/js",
```

Comment:
Instead of introducing new tasks here I think it makes more sense to just re-use the `blocks` tasks and rename them into `public` (as opposed to the `admin` section)?

Reply:
have adjusted this to ensure the blocks' existing code is reused to handle both instead of introducing new one